wavefront_opentracing_python_sdk/reporting/reporter.py

```diff
@@ -0,0 +1,32 @@
+class Reporter(object):
+
+    def __init__(self, source=None):
+        self.source = source
+
+    def report(self, wavefront_span):
+        raise NotImplementedError
+
+    def get_failure_count(self):
+        raise NotImplementedError
+
+    def close(self):
+        raise NotImplementedError
+
+    def send_span(self, sender, wavefront_span):
```

Comment:
If we collapse proxy and direct into a single Reporter as commented above, you can move this method into the `WavefrontReporter` class.